sql-cli/sql_cli/__main__.py

```diff
@@ -203,11 +235,19 @@ def run(
         show_default=True,
     ),
     verbose: bool = typer.Option(False, help="Whether to show verbose output", show_default=True),
+    task_id: str = typer.Option(
+        None,
+        help="Id of the single task to run. If given, only this task of the DAG will run. Otherwise, all tasks of the "
+        "DAG will be run.",
+    ),
+    include_upstream: bool = typer.Option(
```

Comment:
if the user passes `--include_upstream`, we are potentially not showing all the tasks we are skipping. Perhaps we should document this.